### PR TITLE
chore: Update CHANGELOG for release version 0.2.0

### PR DESCRIPTION
**Pull request by dvandra**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+    - Using component-analysis batch API with POST request method. See [#125](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/125)
+    - Update Diagnostic Message. See [#146](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/146)
+    - Show exploitable vulnerability count in Diagnostic Message. See [#137](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/137)
+- enhancement - Upadated Stack Report UI:
+    - Snyk token submission modal to connect Snyk account with Dependency Analytics Report. See [#161](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/161)
+    - View premium fields after connecting Snyk account with Dependency Analytics Report. See [#157](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/157)
+    - Remove Github stats and Licenses details from Security Issues card. See [#166](https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/166)
+- fixes - Can not navigate to synk: Added postMessage to handle url click. See [#403](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/403)
```
Comment (prashbnair) [inline CHANGELOG.md:16]:
```suggestion
- fixes - Can not navigate to Snyk: Added postMessage to handle url click. See [#403](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/403)
```

**Review by prashbnair — APPROVED** [review]

**Comment by arajkumar** [comment]
hacktoberfest-accepted 🥇

**Review by arajkumar — APPROVED** [review]
lgtm

**Comment by dvandra** [comment]
[test]

**Comment by dvandra** [comment]
[test]

**Pull request merged**